.changeset/curvy-bobcats-fry.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This is just personal taste FWIW but I believe we shouldn't forcefully bump all builds, maybe? It'd be cool if there was a way to tell changesets to "ignore" one and only include it when a non-ignored change set was in a release too